openmc/material.py

```diff
@@ -1165,17 +1164,24 @@ def get_activity(self, units: str = 'Bq/cm3', by_nuclide: bool = False,
             of the material is returned as a float.
         """
 
-        cv.check_value('units', units, {'Bq', 'Bq/g', 'Bq/kg', 'Bq/cm3'})
+        cv.check_value('units', units, {'Bq', 'Bq/g', 'Bq/kg', 'Bq/cm3', 'Ci', 'Ci/m3'})
         cv.check_type('by_nuclide', by_nuclide, bool)
 
+        if volume is None:
+            volume = self.volume
+
         if units == 'Bq':
-            multiplier = volume if volume is not None else self.volume
+            multiplier = volume
         elif units == 'Bq/cm3':
             multiplier = 1
         elif units == 'Bq/g':
             multiplier = 1.0 / self.get_mass_density()
         elif units == 'Bq/kg':
             multiplier = 1000.0 / self.get_mass_density()
+        elif units == 'Ci':
+            multiplier = volume / 3.7e10
```

Comment:
maybe worth introducing `BECQUEREL_PER_CURIE` to `data.py` unit conversions section.